openeo/rest/connection.py

```diff
@@ -833,7 +833,7 @@ def load_collection(
         """
         Load a DataCube by collection id.
 
-        :param collection_id: image collection identifier
+        :param id: image collection identifier
```

Comment:
```suggestion
```suggestion
        @deprecated("Use 'id' instead!")
        :param collection_id: image collection identifier
        :param id: image collection identifier
```